src/react/utils.js

```diff
@@ -649,11 +652,13 @@ export function getProperty(
   property: string | SymbolValue
 ): Value {
   if (object instanceof AbstractObjectValue) {
-    let elements = object.values.getElements();
+    let elements = object.values._elements;
```

Comment:
Is this a private field?

Reply:
Yep, but `getElements` triggers an invariant if it's undefined, which is why we can't use it here.

Reply:
Is there a good reason why it triggers an invariant? I suppose somebody put it there to protect against something. Why should we get around that?

Reply:
Fixed. :)